src/sources/paraswap.extralist.json

```diff
@@ -40,8 +40,8 @@
     },
     {
       "chainId": 1,
-      "address": "0x912CE59144191C1204E64559FE8253a0e49E6548",
-      "name": "GHO",
+      "address": "0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f",
+      "name": "Gho Token",
       "symbol": "GHO",
       "decimals": 18,
       "logoURI":
```

Comment:
I wanted to find a png with transparency, but Coingecko [only has jpegs](https://api.coingecko.com/api/v3/coins/gho) :(